examples/web_agent/env.example

```diff
@@ -0,0 +1,6 @@
+# Copy this file to .env and set the appropriate environment variable(s) for your application.
+AIRTOP_API_KEY=""
+requires_login="False"
```

Comment:
not sure why this is an env var - you should have code that utilizes the live view to check if the user is signed in or not

in the typescript CLI examples, we prompt for the profile id, which may or may not mean that the user is signed in

https://github.com/airtop-ai/examples-typescript/blob/main/examples/cli-only-template/src/linkedin-data-extraction.cli.ts

Reply:
I did this to not consume more credits from the user, but I can use the same logic